docs/parser.rst

```diff
@@ -198,10 +189,11 @@ Mandatory message fields, expected by NetlinkSocketBase methods:
 Per-request parsers
 -------------------
 
-Sometimes, it may be reasonable to handle a particular response with a
-specific parser rather than a generic one. An example is
-`IPRoute.get_default_routes()`, which could be slow on systems with
-huge amounts of routes.
+To assign a custom parser to a request/response communication, it is
+enough to provide the parser function to the `NetlinkRequest` object.
+
+An example is `IPRoute.get_default_routes()`, which could be slow on
+systems with huge amounts of routes.
 
 Instead of parsing every route record as `rtmsg`, this method assigns
 a specific parser to its request. The custom parser doesn't parse records
```

Comment:
I never used this advanced filtering, and I think that a real example could be valuable

Reply:
Actually, the whole chapter is about that :) I uploaded the built docs: https://docs.pyroute2.org/parser.html#per-request-parsers

Reply:
Thanks, I will take some time to play with it